lib/libospf.h

```diff
@@ -74,7 +74,7 @@ extern "C" {
 #define OSPF_TRANSMIT_DELAY_DEFAULT         1
 #define OSPF_DEFAULT_BANDWIDTH		 10000	/* Mbps */
 
-#define OSPF_DEFAULT_REF_BANDWIDTH	100000  /* Mbps */
+#define OSPF_DEFAULT_REF_BANDWIDTH 100000 /* Kbps */
```

Comment:
why the change in alignment with the rest of the code.  Please fix

Reply:
ok